index.html

```diff
@@ -23,17 +23,17 @@ <h1>The Yucatáns Peninsula at a Glance</h1>
     <h3>Welcome</h3>
 
       <p>The Yucatán Peninsula is a region of southeastern <a href="https://wikitravel.org/en/Mexico">Mexico</a>, consisting of the Mexican states of Yucatán, Campeche, and Quintana Roo. The Yucatán was the home of the Maya civilization before it was conquered by the Spanish Conquistadors in the 16th century. Much of the population is part or all of Maya descent, and in many places the Maya language is still spoken, usually in addition to Spanish, the main language of business.</p>
-      <p>Until the mid 20th century, most of the Peninsula's trade with the rest of Mexico was by sea, and the culture, cuisine, and traditions developed different flavors from other parts of Mexico. Starting in the late 20th century the Yucatan has become more integrated into Mexico, especially such areas on the Caribbean coast as Cancún and Chetumal, where many people from other parts of the nation have moved to take advantage of the economic opportunities of development. The <a href="https://wikitravel.org/en/Mayan_Riviera">Mayan Riviera</a> stretching south from Cancún has seen the most growth related to tourism.</p>
+      <p>Until the mid 20th century, most of the Peninsula's trade with the rest of Mexico was by sea, and the culture, cuisine, and traditions developed different flavors from other parts of Mexico. Starting in the late 20th century the Yucatán has become more integrated into Mexico, especially such areas on the Caribbean coast as Cancún and Chetumal, where many people from other parts of the nation have moved to take advantage of the economic opportunities of development. The <a href="https://wikitravel.org/en/Mayan_Riviera">Mayan Riviera</a> stretching south from Cancún has seen the most growth related to tourism.</p>
 
 
-    <img src="images/yucatan-map.jpg" alt="a stylized old map of the Yucatan Peninsula">
+    <img src="images/Yucatán-map.jpg" alt="a stylized old map of the Yucatán Peninsula">
     <h3>History & Culture</h3>
```

Comment:
@nhtaylor this image path does not match the name of the image and therefore is not rendering... looks like we have to get rid of the accent?